rgw/v2/lib/resource_op.py

```diff
@@ -221,6 +221,7 @@ def read(self):
         )
         self.frontend = self.doc["config"].get("frontend")
         self.io_op_config = self.doc.get("config").get("io_op_config")
+        self.radoslist_all = self.doc["config"].get("radoslist_all", False)
```

Comment:
according to the below yaml, `radoslist_all` is under `test_ops`. I think this has to be taken form `test_ops` and not from `config`

Reply:
@rakeshgm , changed radoslist_all to be taken from test_ops.

logs at http://magna002.ceph.redhat.com/ceph-qe-logs/madhavi/PRs/radoslist/pacific/review/console.log